visuald/config.d

```diff
@@ -2695,7 +2695,7 @@ class Config :	DisposingComObject,
 			string ext = toLower(extension(fname));
 			if(isIn(ext, ".d", ".ddoc", ".def", ".lib", ".obj", ".o", ".res"))
 				tool = "DMD";
-			else if(ext == "rc")
+			else if(ext == ".rc")

```

Comment:
Ouch. It seems I never noticed because Visual D uses a custom rule to build its resources.